pubtools/pulplib/_impl/client/search.py

```diff
@@ -113,3 +117,34 @@ def field_match(to_match):
         return {"$lt": to_mongo_json(to_match._value)}
 
     raise TypeError("Not a matcher: %s" % repr(to_match))
+
+
+def validate_type_ids(type_ids):
```

Comment:
maybe util.py would be better module. I think this could be reused in different parts of the code

Reply:
This started out a "filter", so it seemed to fit, but became what it is now, which, I agree, would probably be more at home in utils.py.

Reply:
Moving this throws a lot else out of whack. I'll keep trying but I may end up leaving it in search.py.